common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -802,3 +785,44 @@ describe 'Problem', ->
 
       # verify that codemirror textarea has correct `aria-describedby` attribute value
       expect($(CodeMirrorTextArea).attr('aria-describedby')).toEqual('cm-editor-exit-message-101 status_101')
+
+
+  describe 'show answer button', ->
+
+    radioButtonProblemHtml = readFixtures('radiobutton_problem.html')
```

Comment:
Although it seems to use the same logic but i think it's good to test it seperately as there was issue with both type of problems.